addon/instance-initializers/embedded.ts

```diff
@@ -1,6 +1,8 @@
 import ApplicationInstance from '@ember/application/instance'
 
 export function initialize(appInstance: ApplicationInstance): void {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore: https://github.com/typed-ember/ember-cli-typescript/issues/1471
```

Comment:
This comment is not for your actual changes but for the `return` line: (I can't comment on it)

```js
return Object.assign(appConf, embedConf)
```

While we are at it I think we could just change it to:

```js
Object.assign(appConf, embedConf)
```

To make it match the return type `initialize(): void`.